pyflakes/checker.py

```diff
@@ -1231,8 +1231,12 @@ def handleNodeLoad(self, node):
             # the special name __path__ is valid only in packages
             return
 
-        if name == '__module__' and isinstance(self.scope, ClassScope):
-            return
+        if isinstance(self.scope, ClassScope):
+            class C:
+                class_scoped_magic = dir()
```

Comment:
this should probably be at the module scope in pyflakes so it isn't computed every time